cmake/ffmpeg.cmake

```diff
@@ -0,0 +1,9 @@
+find_package(FFMPEG REQUIRED)
```

Comment:
Who provides `FindFFMPEG.cmake`? It's not part of CMake.

Reply:
[Looks like they provide it.](https://github.com/microsoft/vcpkg/blob/master/ports/ffmpeg/FindFFMPEG.cmake.in)